src/main/java/net/goldiriath/plugin/game/skill/menu/SkillLevelingMenu.java

```diff
@@ -0,0 +1,83 @@
+package net.goldiriath.plugin.game.skill.menu;
+
+import net.goldiriath.plugin.Goldiriath;
+import net.goldiriath.plugin.game.item.StaticItem;
+import net.goldiriath.plugin.game.skill.SkillType;
+import net.goldiriath.plugin.player.PlayerData;
+import net.goldiriath.plugin.util.IconMenu;
+import net.pravian.aero.component.PluginComponent;
+import org.bukkit.ChatColor;
+import org.bukkit.entity.Player;
+import org.bukkit.event.inventory.InventoryDragEvent;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SkillLevelingMenu extends PluginComponent<Goldiriath> implements IconMenu.OptionClickEventHandler {
+
+    private final SkillType type;
+
+    public SkillLevelingMenu(Goldiriath plugin, SkillType type) {
+        super(plugin);
+        this.type = type;
+    }
+
+    public void openMenu(Player player) {
+        final IconMenu menu = new IconMenu("Do you want to unlock this skill?", 9, this, plugin);
+
+        menu.setOption(0, StaticItem.MENU_DONE.getStack(), "done");
+        menu.setOption(8, StaticItem.MENU_CANCEL.getStack(), "cancel");
+        menu.open(player);
+    }
+
+    @Override
+    public void onOptionClick(IconMenu.OptionClickEvent event) {
+
+        if (event.getName().equals("done")) {
+            PlayerData data = plugin.pm.getData(event.getPlayer());
```

Comment:
It's probably best if  the logic here is flipped. To illustrate, consider

```java
if (condition) {
  // lots of code
  return;
} else {
  // little code
  return;
}
```
...should probably be flipped to:
```java
if (!condition) {
// little code
return;
}

// lots of code
```

By doing so, you maximize number of lines at a minimal indentation level.